1-Pflichtenheft/pflichtenheft.tex

```diff
@@ -88,6 +88,99 @@ \section{Produktumgebung}
+		\end{enumerate}
+		\item Falls die Wahlen nicht eingeleitet sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Den Kandidat hinzufuegen. Falls diese Option ausgewählt ist, gehen Sie zu 5.6)
+			\item Den Kandidat loeschen Falls diese Option ausgewählt ist, gehen Sie zu 5.7)
+		\end{enumerate}
+		\item Falls die Wahlen schon abgeschlossen sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Berechnen die aktuellen Anzahl der Votes. (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.3)
+			\item Neue Wahlen beginnen (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.2.2)
+		\end{enumerate}
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Zaehlen der Votes}	
+\begin{itemize}
+	\item Ziel: berechnen die Anzahl der von Waehler gegebenen Votes.
+	\item Verwaltung der Wahlen.
+	\item Das System gibt Informationen über jeden Kandidaten (Name, Nachname, Foto des Kandidaten) und die Anzahl der von jedem der Wähler erhaltenen Votes aus. Der Netzwerkadministrator nach der Überprüfung der Informationen geht zurück (siehe 5.2.1 oder 5.2.3). 
+\end{itemize}
+
+\subsection{Beendigung der Wahlen}
+\begin{itemize}
+	\item Ziel: die Wahlen beenden.
+	\item Subsystem: Verwaltung der Wahlen.
+      \item Das System gibt die Absichten des Netzwerkadministrators an, den Abschluss der Wahlen durchzuführen.
+Der Administrator stimmt zu oder lehnt ab.
+	\begin{enumerate}
+		\item Falls Netzwerkadministrator stimmt zu: Das System stoppt den Wahlenprozess. (Dann Uebergang 
+zu 5.2)
+		\item Falls Netzwerkadministrator lehnt ab: Uebergang zu 5.2
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Hinzufuegen des neuen Kandidaten}
+\begin{itemize}
+	\item Ziel: Neue Kandidat in dem Wahlprozess hinzufuegen.
+	\item Subsystem: Verwaltung der Wahlen.
+	\item Das System schlägt vor, Nachname, Vorname, Vatersnamen einzugeben und ein Foto des Kandidaten hochzuladen.
+	\item Der Administrator gibt den Nachnamen, den Namen und den Vatersnamen ein und lädt das Foto des Kandidaten.
+	\item Das System prüft die Daten auf Korrektheit.
+	\begin{enumerate} 
+		\item Wenn die Daten nicht korrekt sind, informiert das System den Benutzer darüber und wechselt zu 5.2.
+		\item Wenn die Daten korrekt sind, fordert das System eine Bestätigung an und fügt, falls bestätigt, der Datenbank Informationen über den neuen Kandidaten hinzu. 
+	\end{enumerate}
+	\item Dann Uebergang zu 5.2.
+\end{itemize}
+
+\subsection{Erstellung der Wahlen}
+\begin{itemize}
+	\item Ziel: “initialize” der Wahlen.
+	\item Verwaltung der Wahlen.
+	\item Das System gibt die Absichten des Netzwerkadministrators an, den Abschluss der Wahlen durchzuführen.
+Der Administrator stimmt zu oder lehnt ab.
+	\begin{enumerate}
+		\item Falls Netzwerkadministrator stimmt zu: Das System stoppt den Wahlenprozess. (Dann Uebergang 
```

Comment:
Das erscheint mir unlogisch: wir sind beim "initializen" (=aufsetzen, erzeugen ?) von einer Wahl. Sie existiert also noch gar nicht in unserem System. Dann kann der Wahlprozess erst recht nicht gestoppt werden weil er nicht einmal angefangen hat.

Reply:
gemacht